tests/Fixer/Alias/ArrayPushFixerTest.php

```diff
@@ -0,0 +1,234 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Fixer\Alias;
+
+use PhpCsFixer\Tests\Test\AbstractFixerTestCase;
+
+/**
+ * @author SpacePossum
+ *
+ * @internal
+ *
+ * @covers \PhpCsFixer\Fixer\Alias\ArrayPushFixer
+ */
+final class ArrayPushFixerTest extends AbstractFixerTestCase
+{
+    /**
+     * @param string      $expected
+     * @param null|string $input
+     *
+     * @dataProvider provideFixCases
+     * @requires PHP 7.0
+     */
+    public function testFix($expected, $input = null)
+    {
+        $this->doTest($expected, $input);
+    }
+
+    public function provideFixCases()
```

Comment:
I think some tests are missing regarding priority: if the second argument of `array_push` contains an operator with less priority than `=`, then [the result is different](https://3v4l.org/trVqp). In such case the expression should be wrapped with parentheses. I think the operators to check are `yield`, `yield from`, `and`, `or` and `xor`.

Reply:
good catch :+1:  added  (after rebase and squash)